internal/scanner/scanner.rl

```diff
@@ -7,7 +7,7 @@ import "github.com/influxdata/flux/internal/token"
 
 	include WChar "unicode.rl";
 
-	newline = '\n' @{ s.f.AddLine(s.f.Base() + fpc) };
```

Comment:
I had removed base when I made this change and forgot to change it back. I'll revert this part.

Reply:
Oh wait, I remember.  This is because the original logic was wrong and only worked because we never added more than one file. The `AddLine` is supposed to be the first _offset_ for where a line _begins_ and this was recording the position of the newline instead of the position of the next line.